### PR TITLE
Manual scaling

### PR DESCRIPTION
**Pull request by jhftrifork**

### CODE REVIEW

**Comment by jrslv** [comment]
I see it's WIP, just letting you know that the scheduler API doesn't work (Connection refused) in this branch. Can't wait to merge it with #331 :-)

**Comment by jhftrifork** [comment]
@jrslv you could take a look at this now. I tried to update your Javascript but can't work out Angular for the life of me, so I think you will have to do it :-)

**Comment by jrslv** [comment]
@jhftrifork the front-end part is now fixed.

**Comment by sadovnikov** [comment]
https://github.com/sadovnikov/elasticsearch/tree/scaling-and-ip-addresses combines this PR and `feature/useIpAddressOption` branch. With that scaling up and down of jar-based ES on `minimesos` works. The only problem noticed is the FAILED status of scaled down executors. Looks good :+1:

File: scheduler/src/main/java/org/apache/mesos/elasticsearch/scheduler/cluster/TaskReaper.java [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+                List<Protos.TaskInfo> taskList = clusterState.getTaskList();
+                int size = taskList.size();
+                numToKill = Math.min(numToKill, size);
+                for (int x = 0; x < numToKill; x++) {
+                    int taskIndex = size - 1 - x;
+                    LOGGER.debug("Killing task index " + taskIndex);
+                    Protos.TaskInfo killTaskInfo = taskList.get(taskIndex);
+                    Protos.TaskID killTaskId = killTaskInfo.getTaskId();
+                    LOGGER.debug("Killing task: " + killTaskId);
+                    Protos.Status status = schedulerDriver.killTask(killTaskId);
```
Comment (sadovnikov) [inline scheduler/src/main/java/org/apache/mesos/elasticsearch/scheduler/cluster/TaskReaper.java]: #426 is logged form this line

**Pull request merged**